### DISCUSSION TITLE
Can I disable moving of a plot to a destination?

### DISCUSSION THREAD

**Discussion by augustynr** [swar/Swar-Chia-Plot-Manager#1162] [Q&A]
Hi, I like for the plots to be left in the temporary directory.
I find the process of copping to the destination very slow so I am planning on using the shell.
What would be the best way to accomplish it.
Can I set the destination directory to the temporary directory? Or that will do additional copping of the plot.
thx

**Comment by gacz81** [comment]
Hello, just put the final destination = temporary directory , exactly the same. If you put same disk but other directory it make a copy anyway (not just move). I actually use it that way, then later copy other way.
